internal/internal.go

```diff
@@ -94,7 +94,10 @@ func ReadLinesOffsetN(filename string, offset uint, n int) ([]string, error) {
 	for i := 0; i < n+int(offset) || n < 0; i++ {
 		line, err := r.ReadString('\n')
 		if err != nil {
-			break
+			if err == io.EOF {
+				break
+			}
+			return []string{""}, err
```

Comment:
Hmm so this now errors rather than breaking if the error is something other than EOF? This looks to only be used by the zfs plugin currently. Will this potential start causing additional failures that we should ignore?

Reply:
@powersj 
Both approaches may behave inaccurate if last non-empty lines ends with `eof` instead od `\n`.

I fixed this and simplified. Please take a look.